### DISCUSSION TITLE
pysr.install() not found

### DISCUSSION THREAD

**Discussion by DenisSvirin** [MilesCranmer/PySR#282] [Q&A] [answered]
I followed installation instructions through step 1 and 2. But when I try `python3 -c 'import pysr; pysr.install()'` , it says that there is no install in pysr.
This is what I got in pysr folder:

`__init__.py  __pycache__  export.py  feynman_problems.py  sr.py`

**Comment by MilesCranmer** [comment]
Looks like a very old version. Try `pip install -U pysr`

**Reply by DenisSvirin** [reply]
i did it, but with user
`pip install -U pysr --user`
Then, just copying files from GitHub is probably fastest solution

**Reply by MilesCranmer** [reply]
Not sure I follow. Is it not updating? You could force version with `pip install -U pysr==0.12.0`

**Reply by DenisSvirin** [reply]
The latest version it has - 0.6.0rc1

**Reply by MilesCranmer** [reply, marked as answer]
That version is 2 years old. Seems like an issue with your pip install or python env, the latest version available is 0.12.0: https://pypi.org/project/pysr/#history

**Reply by DenisSvirin** [reply]
Oh, ye ,apparently, the server has old python, thanks
